raiden_libs/properties/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .address import address_property
+
+__all__ = [
```

Comment:
Do we actually need to do list this here?

Reply:
I don't see why not. That's a detail I really don't ponder over, it can be changed/removed once we stabilise the codebase.